### PR TITLE
Present modal when non web3-enabled browser is detected

### PR DESCRIPTION
**Pull request by saturnial**

### CODE REVIEW

**Comment by graemecode** [comment]
Nice!

**Comment by nadavhollander** [comment]
@saturnial Mobile looks a wee bit funky: https://cl.ly/0z2L2O320W2h 
The modal could use some styling / design love as well -- looks a bit ugly right now.  LMK if you want to bounce some basic designs back n forth -- I can whip these things up fairly quickly

File: src/components/MobileModal/MobileModal.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    render() {
+        const content = (
+            <div>
+                <span>
+                    Dharma Plex is a web3-enabled application. You will need to use one of the below
+                    web3-enabled browsers in order to properly interact with this site.
+                </span>
+                <ul>
+                    <li>
+                        <A href="https://www.toshi.org/" target="_blank">
```
Comment (nadavhollander) [inline src/components/MobileModal/MobileModal.tsx]: The styling of the links you've included make them black -- you'll need to mess with the CSS to make sure they appear clearly as links and not just text (as I thought they were when I first saw them)

File: src/components/MobileModal/MobileModal.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                            Toshi
+                        </A>
+                    </li>
+                    <li>
+                        <A href="https://www.cipherbrowser.com/" target="_blank">
+                            Cipher
+                        </A>
+                    </li>
```
Comment (nadavhollander) [inline src/components/MobileModal/MobileModal.tsx]: Let's add MetaMask in here and distinguish between mobile wallets / browser extensions.

**Comment by nadavhollander** [comment]
@saturnial Proposed mock: 

modalmock

**Pull request merged**